local/README.md

```diff
@@ -83,6 +83,14 @@ The Flow UI should open in your browser automatically.
 
 To login locally, you need to use the "magic link" method. Oauth does not work locally. You can enter any email address you like, and all emails will be sent to [http://localhost:5434/monitor]. After it says "email sent", go to that address and click the link in the email to complete the login process.
 
+### Data-plane-gateway TLS certificates
```

Comment:
I read about this tool recently that manages creating certs and installing them in local browser stores https://github.com/FiloSottile/mkcert. Might be worth using this?

Reply:
The implemented solution seems very satisfactory, though, and has the benefits of simplicity and not pulling in yet another dependency to learn about and maintain.